stories/GetStarted.mdx

```diff
@@ -1,10 +1,11 @@
-import { Meta, ColorPalette, ColorItem } from '@storybook/blocks'
+import { Markdown, Meta, ColorPalette, ColorItem } from '@storybook/blocks'
 import aliasColors from '@/colors'
 import { colors } from '@/colors/base'
 import { ColorWrapper } from './colorUtils.tsx'
+import ReadMe from '../README.md?raw'
 
 <Meta title="Get Started" />
 
-<h1>Get Started</h1>
-
-<p>Welcome to Quantum. This guide will help you to get up and running in no time.</p>
+<Markdown>
+  {ReadMe.replace(/!\[CI\]\(.*\)\s*\[!\[Figma\]\(.*\)\]\(.*\)/, '')}
```

Comment:
will the badges not work within storybook?

Reply:
<img width="821" alt="image" src="https://github.com/nearform/quantum/assets/1158133/aa868269-4dee-43e7-b4d4-8f416f1b9225">
This is what caught my attention. Should I look at a solution for having both in there?

Reply:
ideally yes, weird that the firma badge doesn't reneder though, it looks like valid markdown that storybook should be able to render

Reply:
@simoneb I tried latest version of Markdown from storybook/blocks but it didn't work either so I used react-markdown (this was my initial lib anyway) which seems to be rendering it ok (does a new line not space, but that works I. guess)